components/FlexibleHeader/src/MDCFlexibleHeaderView.m

```diff
@@ -822,7 +822,15 @@ - (void)fhv_accumulatorDidChange {
 
 - (void)fhv_updateLayout {
   if (!_trackingScrollView) {
-    return;
+    // Even if we're not tracking a scroll view, our minimumHeight might have changed due to new
+    // safe area insets.
+    if (_minimumHeight > 0 && self.bounds.size.height != _minimumHeight) {
```

Comment:
frame and bounds may have negative values for size.height and size.width.
Use CGRectGetHeight(self.frame) and other CGRectFunc() instead of accessing the struct directly.

Reply:
Done